src/lib.rs

```diff
@@ -88,6 +88,10 @@
 #![cfg_attr(any(proc_macro_span, super_unstable), feature(proc_macro_span))]
 #![cfg_attr(super_unstable, feature(proc_macro_def_site))]
 #![cfg_attr(docsrs, feature(doc_cfg))]
+#![cfg_attr(
+    all(procmacro2_semver_exempt, feature = "proc-macro"),
+    feature(proc_macro_value)
```

Comment:
It is not worth introducing the use of an unstable feature for this API. That makes sense for things like Span::def_site and Literal::subspan where a stable implementation is impossible. But for the methods in this PR they should just follow the stable "fallback" implementation in both cases, without being coupled to the API in nightly.